core/cmd/local_client.go

```diff
@@ -73,6 +81,25 @@ func loggedStop(app chainlink.Application) {
 	logger.WarnIf(app.Stop())
 }
 
+func checkFilePermissions(directory string) error {
+	err := filepath.Walk(directory,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			perms := info.Mode()
+			// bitwise & used to check if file has any invalid permissions
+			if perms&invalidPerms != 0 {
+				return fmt.Errorf("%s has overly permissive file permissions, %s", path, perms.String())
```

Comment:
```suggestion
				return fmt.Errorf("%s has overly permissive file permissions, %s. Permissions should be 600.", path, perms.String())
```

Reply:
:+1: but @kaleofduty this was mentioned above, and the permissions should be taken from the constant.

Reply:
permissions shouldn't necessarily be 0600. they could be anything 0100 - 0700 so long as the last two digits are 0s.